### PR TITLE
[CS-2229] Fix max amount calc issue in depot sending

### PR DESCRIPTION
**Pull request by topwebtek7**
### Description

- [x] Completes #CS-2229

### Checklist

- [x] All UI changes have been tested on a small device

### Screenshots

Oct-28-2021 16-45-26

### CODE REVIEW

File: cardstack/src/screens/SendSheetDepot/useSendSheetDepotScreen.ts [hunk 1]
```diff
@@ -99,18 +99,17 @@ export const useSendSheetDepotScreen = () => {
       setGasEstimatedFee(gasFeeInUsd);
 
       // Calculate maxBalance
-      const currentBalanceWei = new BigNumber(selected?.balance?.wei || '0');
-      const gasEstimateWei = new BigNumber(gasEstimate);
+      const currentBalanceWei = new BN(selected?.balance?.wei || '0');
```
Comment (danibonilha) [inline cardstack/src/screens/SendSheetDepot/useSendSheetDepotScreen.ts:102]: Just for context: 
What was the error ? Why do we need to switch BigNumber to BN ?

File: cardstack/src/screens/SendSheetDepot/useSendSheetDepotScreen.ts [hunk 2]
```diff
@@ -4,10 +4,10 @@ import {
 } from '@cardstack/cardpay-sdk';
 import { useRoute } from '@react-navigation/native';
 import { captureException } from '@sentry/react-native';
-import BigNumber from 'bignumber.js';
 import { get, isEmpty, isString } from 'lodash';
 import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
 import Web3 from 'web3';
+import BN from 'bn.js';
```
Comment (danibonilha) [inline cardstack/src/screens/SendSheetDepot/useSendSheetDepotScreen.ts:10]: It seems there's no change on `package.json` so where is this lib coming from ?

File: cardstack/src/screens/SendSheetDepot/useSendSheetDepotScreen.ts [hunk 3]
```diff
@@ -292,7 +291,7 @@ export const useSendSheetDepotScreen = () => {
 
       navigate(Routes.PROFILE_SCREEN);
     } catch (error) {
-      const errorMessage = error.toString();
+      const errorMessage = (error as any).toString();
```
Comment (danibonilha) [inline cardstack/src/screens/SendSheetDepot/useSendSheetDepotScreen.ts:294]: Why do we need to cast this ? I thought it was already any 🤔

File: cardstack/src/screens/SendSheetDepot/useSendSheetDepotScreen.ts (hunk 2 above)
Comment (topwebtek7) [reply cardstack/src/screens/SendSheetDepot/useSendSheetDepotScreen.ts:10]: this lib comes from sdk

File: cardstack/src/screens/SendSheetDepot/useSendSheetDepotScreen.ts (hunk 1 above)
Comment (topwebtek7) [reply cardstack/src/screens/SendSheetDepot/useSendSheetDepotScreen.ts:102]: Web3.utils.fromWei needs BN, BigNumber to BN was tricky with long digit numbers(i.e. 7.90992638554142277326e+21) and that was the reason of the issue.. 
I posted in ticket as well but this was the error:
`[Error: [number-to-bn] while converting number "7.90992638554142277326e+21" to BN.js instance, error: invalid number value. Value must be an integer, hex string, BN or BigNumber instance. Note, decimals are not supported.]`

File: cardstack/src/screens/SendSheetDepot/useSendSheetDepotScreen.ts (hunk 3 above)
Comment (topwebtek7) [reply cardstack/src/screens/SendSheetDepot/useSendSheetDepotScreen.ts:294]: this was the issue with TS in my vsc
image

**Review by danibonilha — APPROVED** [review]
🚀

**Pull request merged**